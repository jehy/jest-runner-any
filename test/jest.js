'use strict';

it('Works when it has only passing tests', () => {
  return expect({a: 1}).toMatchSnapshot();
});
