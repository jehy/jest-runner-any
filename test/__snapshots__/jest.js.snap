// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works when it has only passing tests 1`] = `
Object {
  "a": 1,
}
`;
